tests/pass0/t03_drop.test

```diff
@@ -0,0 +1,19 @@
+discard """
+  output: "(Done)"
+"""
+(TypeDefs
+  (Int 8)
+  (ProcTy (Void))
+)
+(GlobalDefs)
+(ProcDefs
+  (ProcDef (Type 1) (Locals)
+    (Continuations
+      (Continuation (Params) 0
+        (Drop (IntVal 8))
```

Comment:
Yes, but from how many meters up? ;)

Reply:
Since `Drop` makes the value "disappear", from pretty high up, I'd say. :D